webrtc.html

```diff
@@ -1862,6 +1868,39 @@
         </dl>
       </section>
     </section>
+    <section>
+      <h3>Session Negotiation Model</h3>
+      <p>Many changes to the PeerConnection state will require communication with
+        the remote side via the signaling channel. The app can be kept informed
+        as to when it needs to do signaling by listening to the
+        negotiationneeded event.</p>
+      <p>If an operation is performed on a RTCPeerConnection that requires signaling,
+        the connection will be marked as needing negotiation. Examples of such
```

Comment:
I tend to agree with Justin here.  If there is a browser-driven scenario that is well-enough defined, then I'm not opposed to this, but I think that the situation is complex enough to warrant this sort of help.  After all, there are some operations on local tracks that don't result in a need for negotiation and having application developers understand the minute detail of what needs what would be doing them a disservice.